Examples/ByFeature/FunctionalInjection/Shopping.feature

```diff
@@ -1,7 +1,19 @@
 ﻿Feature: Shopping
 
-Scenario: Shopping in a store
-Given I have 100 dollars in my wallet
-When I buy an item for 5 dollars
-And I buy an item for 10 dollars
-Then My wallet contains 85 dollars
\ No newline at end of file
+This sample demonstrates the usage of Tables, Lists and Doc Strings with Functional Injection.
+
+Scenario: Making a purchase
+  Given the 2018 product catalog:
```

Comment:
It would be nice to have it in some state that noone starts thinking "why there is such parameter". You added parameters which does not have any exact meaning. However, it will make the test somewhat harder, so I would prefer to rename the Shopping.feature to Wallet.feature and add a Shopping.feature with your scenario which would demonstrate more advanced techniques to use.

I can imagine that you will have background of the feature and using multiple given steps you will prepare catalogs for individual years. In the when sentence you will specify the date when you are placing the order which selects the catalog and stores the receipt with correct date. And last, in the then clause you will use the date to retrieve the receipt and verify it.

Reply:
Thanks for the suggestions. I've re-worked the example to utilise the parameters in each step and renamed the original Shopping.feature to Wallet.feature.